server/views/pages/goals/goalsPage.njk

```diff
@@ -0,0 +1,101 @@
+{% extends "../index.njk" %}
```

Comment:
I'm just thinking about the name of this page (plus the corresponding ts pages in the e2e tests)...would `vc2GoalsPage.njk` be better? It's possible we might want to add a page for PLP goals in the future and it also makes it clear that this page is specifically for VC2.

Reply:
As discussed on Slack we agreed to rename this